rootfs/mkrootfs_ubuntu.sh

```diff
@@ -0,0 +1,180 @@
+#!/bin/bash
```

Comment:
I believe `mkrootfs_debian.sh` can handle creating jammy distro just fine. I don't think this script is needed.